src/sessions.ts

```diff
@@ -973,7 +970,18 @@ export class ServerSession {
   isDirty: boolean;
 
   /** @internal */
-  constructor() {
+  constructor(
+    cloned?: { id: ServerSessionId; lastUse: number; txnNumber: number; isDirty: boolean } | null
```

Comment:
Why type this separately vs just using the ServerSession type?

Reply:
Good catch, fixed.

I did this manual type b/c originally I was modifying clone, and passed in the plain object we had in there. But once I figured the helper didn't have much value I merged the logic into the ctor.